src/Symfony/Component/HttpKernel/Event/ExceptionEvent.php

```diff
@@ -31,14 +31,17 @@ final class ExceptionEvent extends RequestEvent
 {
     private \Throwable $throwable;
     private bool $allowCustomResponseCode = false;
-    private bool $isKernelTerminating = false;
 
-    public function __construct(HttpKernelInterface $kernel, Request $request, int $requestType, \Throwable $e, bool $isKernelTerminating = false)
-    {
+    public function __construct(
+        HttpKernelInterface $kernel,
+        Request $request,
+        int $requestType,
+        \Throwable $e,
+        private bool $isKernelTerminating = false,
```

Comment:
maybe not worth it here

Reply:
Looking at other similar changes where arguments a mix of CPP + non-CPP, there's no clear cut so I'm 🤔
Any other opinion?

Reply:
I like clear rules, something that we could let a tool enforce in the future. My proposal would be, if CPP is possible, do it.

Especially in complex constructors, CPP helps me because it removes all the standard cases (assign `$property` to `$this->property`). Less boilerplate to process for my brain.

Reply:
I agree with Alexander. Nicolas is right ... but having a no-exception rule for this change will be better in the long term.